modules/javafx.controls/src/main/java/javafx/scene/control/PopupControl.java

```diff
@@ -213,18 +213,6 @@ public PopupControl() {
         // a reference to the old value.
         private Skin<?> oldValue;
 
-        @Override
```

Comment:
This logic was introduce to deal with skins set from the css JDK-8096194

Are you sure it will still work?  Do we need to test this scenario explicitly?

Reply:
No, the `skin`/`skinProperty` logic was there before, just moved.
It was added in https://bugs.openjdk.org/browse/JDK-8127070, 
with the comment from `David Grieve`: `The code for handling the skin property in PopupControl needs to look like the code in Control.`

So they basically synchronized the whole skin/skin name API to `PopupControl`.